src/scripts/farming/Farming.ts

```diff
@@ -1026,8 +1025,6 @@ class Farming implements Feature {
 
         const amount = plot.harvestAmount();
 
-        GameHelper.incrementObservable(App.game.statistics.totalBerriesHarvested, amount);
-        GameHelper.incrementObservable(App.game.statistics.berriesHarvested[plot.berry], amount);
```

Comment:
Do you think it would it be worth having a "totalManualPlotsHarvested" statistic or similar?